pkg/xds/generator/testdata/profile-source/3-envoy-config.golden.yaml

```diff
@@ -177,14 +177,13 @@ resources:
                 httpFilters:
                   - name: envoy.router
                 routeConfig:
-                  name: inbound:backend
+                  name: 'inbound:'
```

Comment:
something is wrong, why this is empty now?

Reply:
Was a mistake in the test. Thanks.